Makefile

```diff
@@ -147,14 +147,12 @@ properclean: ## Completely clean the repository's build artifacts.
 
 .PHONY: git2go
 git2go: $(VENDORDIR)/libgit2/git2go/static-build/install/lib/pkgconfig/libgit2.pc
-	$(GO) install -tags static github.com/libgit2/git2go/v31/...
+	$(GO) install -tags static github.com/libgit2/git2go/v34/...
 
 $(VENDORDIR)/libgit2/git2go/static-build/install/lib/pkgconfig/libgit2.pc: $(VENDORDIR)/libgit2/git2go/vendor/libgit2
 	$(MKDIR) -p $(VENDORDIR)/libgit2/git2go/static-build/build
 	$(MKDIR) -p $(VENDORDIR)/libgit2/git2go/static-build/install
 	(cd $(VENDORDIR)/libgit2/git2go/static-build/build && $(CMAKE) \
-		-DTHREADSAFE=ON \
-		-DBUILD_CLAR=OFF \
```

Comment:
I removed these because they aren't used:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_CLAR
    THREADSAFE
```